fusedrug/utils/cluster/cluster_using_mmseqs.py

```diff
@@ -0,0 +1,156 @@
+from fuse.data.ops import get_function_call_str
+import shutil
+import hashlib
+import subprocess
+from fuse.utils.file_io import save_text_file_safe, read_text_file
+import os
+
+def cluster(force_rebuild:bool = False, **kwargs):
+    """
+    A wrapped around cluster_impl() to allow caching 
+    see cluster_impl doc for details
+    
+    Args:
+        force_rebuild: rebuilds the data even if the hash indicates that it was already cached.
```

Comment:
document here kwargs (cluster_impl) arguments.

Reply:
Ii will create duplication, which can easily go out of sync. I'll try to find some way to have the top level function have the doc, while still supporting cache/hash.

Reply:
done